src/main/java/com/softdevsix/domain/entity/file/File.java

```diff
@@ -0,0 +1,13 @@
+package com.softdevsix.domain.entity.file;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class File {
+  @GeneratedValue(strategy = GenerationType.UUID)
```

Comment:
According to the information you posted in the PR, your scope does not require you to configure JPA, right?

The reason I say this is because the application throws JPA out of configuration exceptions:
![Screenshot from 2024-11-28 21-08-51](https://github.com/user-attachments/assets/19ebd2a0-3bc8-43a7-b43c-6110769156c7)

Reply:
It's updated on the new PR